middleware/reverse/README.md

```diff
@@ -0,0 +1,65 @@
+# reverse
+
+The *reverse* middleware allows CoreDNS to respond dynamic to an PTR request and the related A/AAAA request.
+
+## Syntax
+
+~~~
+reverse NETWORK {
+    [template TEMPLATE]
+    [domain DOMAIN]
+    [ttl TTL]
+}
+~~~
+
+* **NETWORK** is the CIDR formatted network to respond on.
+* `template` inject the ip to an template for the hostname. Defaults to "ip-{ip}".
+    v4 replaces the "." with an "-". 10.1.1.1 results in "10-1-1-1"
+    v6 removes the ":" and fills the zeros. "ffff::ffff" results in "ffff000000000000000000000000ffff" 
+* `domain` defaults to the listen zone.
+* `ttl` defaults to 60
+
+Note that you have to make sure that this middleware will get actual queries for the
+following zones: `in-addr.arpa`, `ip6.arpa`.
```

Comment:
so to makes this work your stanza should have
`in-addr.arpa ip6.arpa compute.internal {`

And the zones for this stanza can be multiple, but `domain` can only hold one?